Gruntfile.js

```diff
@@ -135,6 +135,8 @@ module.exports = function (grunt) {
                             '!thirdparty/CodeMirror2/mode/**/*.html',
                             '!thirdparty/CodeMirror2/**/*test.js',
                             'thirdparty/CodeMirror2/theme/{,*/}*',
+                            'thirdparty/filer-dialogs/bower_components/font-awesome/css/font-awesome.min.css ',
```

Comment:
One thing I'm wondering is whether this is a good idea at all.  If we load it from here, and another extension wants to use it (HTMLHinter does atm), we'll end up loading it twice.  Should we load it from https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.min.css or something?